tests/bcl-test/BCLTests/TestRunner.NUnit/SimpleResultWriter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.IO;
+using System.Net.Mime;
+using NUnit.Framework.Api;
+using NUnit.Framework.Internal;
+
+namespace  Xamarin.iOS.UnitTests.NUnit {
+	/// <summary>
+	/// Simple writer that outputs the results as normal text.
+	/// </summary>
+	public class SimpleResultWriter {
+
+
```

Comment:
Is the output of this meant to be parsed by a machine or a human?
If it is meant to be a machine, can you include a document on the format and/or consider another common format (json, xml, etc)?

Reply:
yeah, I'm going to remove the writer since I found a nicer way to deal with this. :)